ops/ccs-ops-misc/load_test/v1/patient_test_id.py

```diff
@@ -1,43 +1,17 @@
-import urllib3
-import common.config as config
-import common.data as data
-import common.errors as errors
-import common.test_setup as setup
-import common.validation as validation
-from locust import HttpUser, task, events
+from common.BFDUserBase import BFDUserBase
+from common.url_path import create_url_path
+from common.validation import SLA_PATIENT
+from locust import task
 
-server_public_key = setup.loadServerPublicKey()
-setup.disable_no_cert_warnings(server_public_key, urllib3)
+class BFDUser(BFDUserBase):
+    DATA_REQUIRED = [ 'BENE_IDS' ]
+    SLA_BASELINE = SLA_PATIENT
 
-bene_ids = data.load_bene_ids()
-client_cert = setup.getClientCert()
-setup.set_locust_env(config.load())
 
-class BFDUser(HttpUser):
     @task
-    def patient_by_id(self):
-        if len(bene_ids) == 0:
-            errors.no_data_stop_test(self)
```

Comment:
I noticed this got abstracted; what happens now when a test runs out of benes? The way we handle running out of data is probably going to want to be different depending on if we're running the regression suite (where we probably want to continue with the other endpoints?) and a single test (where we want to stop the test).

Reply:
I just tested this, and it looks like the tests continue for the allotted time without sending any requests. Obviously not really what we want. I'll correct that.